decentraland/unity-renderer
# feat: passport clickable coordinates

## What does this PR change?

`Fixes #4175 `

Adds the possibility of having coordinate links into the passport description.
Includes a refactor to improve the communication with the scene metadata, better management of responsibilities in the GoToPanel.

## How to test the changes?

1. Go to: https://play.decentraland.org/?explorer-branch=feat/passport-clickable-coordinates
2. Open your profile modification window
3. Add parcel coordinates, for example: `This is my custom scene: 34,12 . This is my favorite 100,100`
4. Open your passport
5. Check that the description contains the coordinate links
6. Click on any of them. A popup should be displayed including the scene information
7. Write a coordinate in the chat. Click it. It should keep working as expected

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
